dvc/command/daemon.py

```diff
@@ -64,5 +65,7 @@ def add_parser(subparsers, parent_parser):
         description=DAEMON_ANALYTICS_HELP,
         help=DAEMON_ANALYTICS_HELP,
     )
-    daemon_analytics_parser.add_argument("target", help="Analytics file.")
+    daemon_analytics_parser.add_argument(
+        "target", help="Analytics file.", choices=choices.Required.FILE
```

Comment:
Is there a way to tell shtab that it shouldn't complete these commands? `daemon` is hidden (kinda) and shouldn't be completed.

Reply:
I could add a `choices.Suppress`

Reply:
@casperdcl For the whole `daemon` subcommand, right? Yeah, that might be alright.

Reply:
ah not for the whole subcommand - just for this argument.

Excluding an entire subcommand I think would make more sense higher-up... Something like:

```python
shtab.complete(parser, skip=["daemon"], ...)
```

Reply:
@casperdcl That seems strange though :D Maybe there is something we can do in a more explicit way? `choices` worked out very nice, so I wonder if there is some appropriate trick here as well.

Reply:
Not a deal breaker or anything, just wondering.

Reply:
I think the existing `add_help=False` argument for subparsers can actually be used to skip instead. Released in `shtab==1.0.0` :)